src/message/MentionedUserNotSubscribed.js

```diff
@@ -0,0 +1,71 @@
+/* @flow strict-local */
+
+import React, { Component } from 'react';
+import { View, TouchableOpacity } from 'react-native';
+
+import type { Auth, Stream, Dispatch, Narrow, UserOrBot, GetText } from '../types';
+import { withGetText } from '../boot/TranslationProvider';
+import { connect } from '../react-redux';
+import * as api from '../api';
+import { ZulipButton, RawLabel } from '../common';
+import { getAuth, getStreamInNarrow, getUserForId } from '../selectors';
+import styles from '../styles';
+
+type SelectorProps = {|
+  auth: Auth,
+  stream: { ...Stream },
```

Comment:
Prefer exact object types wherever possible; see discussion at #3452.

Reply:
Changed to `Subscription | {| ...Stream, in_home_view: boolean |}`, directly copied from the selector.